crowbar_framework/Gemfile

```diff
@@ -15,7 +15,7 @@
 # limitations under the License.
 #
 
-source "https://rubygems.org"
+source "http://rubygems.org"
```

Comment:
Switching to http is not a real issue and not too bad as we never send any sensitive informations. It grabs only some additional rubygems used for the development environment as you have dropped the development and test dependencies from the devel subpackage.